tests/Stripe/HttpClient/CurlClientTest.php

```diff
@@ -297,4 +297,24 @@ public function testResponseHeadersCaseInsensitive()
         $this->assertNotNull($headers['request-id']);
         $this->assertEquals($headers['request-id'], $headers['Request-Id']);
     }
+
+    public function testSetRequestStatusCallback()
+    {
+        try {
+            $called = false;
+
+            $curl = new CurlClient();
+            $curl->setRequestStatusCallback(function ($rbody, $rcode, $rheaders, $errno, $message, $shouldRetry, $numRetries) use (&$called) {
```

Comment:
can we add a test that each arg is roughly as desired?

Reply:
👍